x/clp/client/cli/tx.go

```diff
@@ -133,9 +133,17 @@ func GetCmdRemoveLiquidity(cdc *codec.Codec) *cobra.Command {
 			externalAsset := types.NewAsset(viper.GetString(FlagAssetSourceChain),
 				viper.GetString(FlagAssetSymbol),
 				viper.GetString(FlagAssetTicker))
-			wBasis := viper.GetInt(FlagWBasisPoints)
-			asymmetry := viper.GetInt(FlagAsymmetry)
+			wb := viper.GetString(FlagWBasisPoints)
+			as := viper.GetString(FlagAsymmetry)
 			signer := cliCtx.GetFromAddress()
+			wBasis, ok := sdk.NewIntFromString(wb)
+			if !ok {
```

Comment:
It would be nice to have a test that triggers these conditions.

Reply:
I think the demo.sh does trigger these sets of conditions. Its a shell script though , so it might not show up in go test -cover .

Reply:
Hey @Jedi2002 can you please confirm a unit test in go code has been written to close off this comment?

Reply:
Yes ,unit tests were added

Reply:
Nice one thanks.